docs/spec/v1/gitrepositories.md

```diff
@@ -195,8 +197,10 @@ stringData:
     -----BEGIN OPENSSH PRIVATE KEY-----
     ...
     -----END OPENSSH PRIVATE KEY-----
+  #github.com usage example : base64 below the correct line (github.com ecdsa-sha2-nistp256) of this command output: ssh-keyscan github.com
+  #gitlab.com usage example : base64 below the correct line (gitlab.com ecdsa-sha2-nistp256) of this command output: ssh-keyscan gitlab.com
   known_hosts: |
-    github.com ecdsa-sha2-nistp256 AAAA...
+    Z2l0bGFiLmNvbSBlY2Rz...
```

Comment:
Why is the known hosts base64 encoded? We use `stringData` here so it should be in clear text.

Reply:
You're right I reverted this line. The idea here was just to add a little comment to explain how to find this necessary value quickly and effectively.